db/schema.rb

```diff
@@ -2,8 +2,8 @@
 # of editing this file, please use the migrations feature of Active Record to
```

Comment:
This file should be reverted too. This is an old schema from rails 6.0, not the new 6.1 one.